### PR TITLE
Closes AgileVentures/MetPlus_tracker#254

### PR DESCRIPTION
**Pull request by joaopapereira**

### CODE REVIEW

File: spec/models/job_seeker_spec.rb [hunk 1]
```diff
@@ -42,58 +42,52 @@
...
+      end
     end
-    it 'returns a job developer for a given job seeker' do
-      expect(dave.job_developer).to eq(jd_person)
+    describe '#assign_job_developer' do
+      it 'success' do
+        bob.assign_job_developer(jd_person, agency)
+        expect(bob.job_developer).to eq(jd_person)
+      end
+      it 'not a case manager' do
```
Comment (patmbolger) [inline spec/models/job_seeker_spec.rb]:
```
it 'not a job developer' do
```

File: spec/models/job_seeker_spec.rb [hunk 2]
```diff
@@ -42,58 +42,52 @@
...
+      end
     end
-
-    it 'finds the agency relation for a job seeker in a given role' do
-      expect(dave.find_agency_person(:JD)).to eq(jd_person)
+    describe '#job_developer' do
+      it 'success' do
+        expect(dave.job_developer).to eq(jd_person)
+      end
+      it 'no case manager' do
```
Comment (patmbolger) [inline spec/models/job_seeker_spec.rb]:
```
it 'not a case manager' do
```

**Comment by patmbolger** [comment]
PR review complete - minor labelling issues noted.

**Comment by joaopapereira** [comment]
Corrected all the PR comments

**Pull request merged**